.github/flakybot.yaml

```diff
@@ -0,0 +1 @@
+issuePriority: p2
```

Comment:
Please add a license header similar to
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/flakybot.yaml